pages/task/read/views/models/place.jsx

```diff
@@ -97,6 +100,22 @@ export default function Playback({ task, values, allowSubmit }) {
           setDirty={setDirty}
         />
       </StickyNavAnchor>
+    ),
+    (
+      <StickyNavAnchor id="conditions" key="conditions">
+        <h2><Snippet>conditions.title</Snippet></h2>
+        <Conditions
+          conditions={taskData.conditions ? taskData.conditions : establishment.conditions }
+          reminders={taskData.conditions ? [JSON.parse(taskData.reminder)] : establishment.reminders}
```

Comment:
Similar to my previous comment. Is parse necessary?

Also should the logic on populating the Condition components reminders property be consistent?